packages/playwright-core/src/server/supplements/har/harTracer.ts

```diff
@@ -49,7 +50,8 @@ export class HarTracer {
   private _started = false;
   private _entrySymbol: symbol;
 
-  constructor(context: BrowserContext, delegate: HarTracerDelegate, options: HarTracerOptions) {
+  constructor(apiRequest: APIRequestContext, context: BrowserContext | undefined, delegate: HarTracerDelegate, options: HarTracerOptions) {
```

Comment:
Should we make it `context: BrowserContext | APIRequestContext` instead? This would make it clear that api request context must belong to the browser context.

Reply:
I can do that, I didn't like instanceof checks that we'd have to introduce in some places.

Reply:
Updated.